lib/v2/screens/profile_screens/security/components/custom_switch.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/constants/app_colors.dart';
+
+class CustomSwitch extends StatefulWidget {
```

Comment:
I am not a fan of this custom switch. 
I think we should use flutters native switch. 

Please use flutter native. Ill ask Jason to update designs to use flutter native.

Reply:
Ok I'll make the chanche